### DISCUSSION TITLE
Why is the Windows Package Manager slow on my machine?

### DISCUSSION THREAD

**Discussion by zjarlin** [microsoft/winget-cli#1108] [Q&A]
winget is too slow

**Comment by Masamune3210** [comment]
gonna offer help to fix it? or just whine

**Comment by zjarlin** [comment]
When I type winget search or install

/

It keeps spinning around and there's no response
发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

发件人: ***@***.***>
发送时间: 2021年6月4日 12:43
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [microsoft/winget-cli] winget (#1102)


gonna offer help to fix it? or just whine

―

**Comment by TheJaysH** [comment]
Works great on my PC, and i dont have the best internet. What speeds do you get?

**Comment by jedieaston** [comment]
Can you post the output of `winget --info`? (If it doesn't work either, I can give you some other commands to get the same info).
